src/components/App.tsx

```diff
@@ -7757,6 +7774,30 @@ class App extends React.Component<AppProps, AppState> {
     this.setState({ suggestedBindings });
   }
 
+  public setSelection(elements: readonly NonDeletedExcalidrawElement[]) {
+    const selectedElementIds: { [id: string]: true } = {};
+    const selectedGroupIds: { [id: string]: true } = {};
+
+    elements.forEach((ele) => {
+      if (ele.groupIds.length) {
+        selectedElementIds[ele.id] = true;
```

Comment:
checking for `!isBoundToContainer(ele)` should be done here as well. In other words, the `selectedElementIds[ele.id] = true` should be assigned only once at the top, irrespective of groups